terraform/modules/helm/druid_raw_cluster/druid-raw-cluster-helm-chart/values.yaml

```diff
@@ -27,7 +31,10 @@ druid_storage_directory: "/data"
 # AWS S3 Details
 s3_access_key: ""
 s3_secret_key: ""
-s3_bucket: "obsrv-sb-dev-165052186109"
+s3_bucket: "" # s3 container name
```

Comment:
Please confirm whether all these default values are mentioned already in the doc or not.